docs/build/guides/dapps/state-archival.mdx

```diff
@@ -0,0 +1,436 @@
+
+Some state archival terminology we will be using in this guide are described in the [state archival section](/docs/learn/encyclopedia/storage/state-archival#terms-and-semantics).
+
+## State archival basics
+
+Stored data in Stellar smart contracts can be categorized into three types:
+
+1. [`Persistent`](/docs/learn/encyclopedia/storage/state-archival#persistent): long-term storage for important data.
+2. [`Temporary`](/docs/learn/encyclopedia/storage/state-archival#temporary): short-term storage for transient data.
+3. [`Instance`](/docs/learn/encyclopedia/storage/state-archival#instance): storage tied to the contract instance itself.
+
+The TTL (Time to Live) is the number of ledgers between the current ledger and the final ledger for which the data can still be accessed.
+
+State archival in Stellar dapps is a crucial consideration for managing the lifecycle and accessibility of contract data and code on the blockchain. It revolves around the concept of TTL, where each data entry, whether `Persistent`, `Temporary`, or `Instance` has a countdown to inaccessibility. When an entry's TTL reaches zero, it becomes either archived or deleted, depending on its type. `Temporary` data is permanently removed, while `Persistent` and `Instance` data become inaccessible but can be restored.
+
+This system allows for efficient blockchain resource management by enabling temporary data to expire and be removed while providing mechanisms to keep important data accessible long-term. Developers must actively manage their data's lifecycle by extending TTLs and restoring archived data when necessary. The process applies not only to contract data but also to contract instances and their associated WebAssembly (WASM) code. Different storage types have varying costs and archival behaviors, allowing developers to optimize based on their specific needs. Ultimately, state archival requires careful consideration in dapp design and ongoing maintenance to ensure continued functionality.
+
+### What can be archived?
+
+- **Contract data**:
+
+  - `Persistent` and `Instance` data entries can be archived when their TTL reaches zero.
+  - `Temporary` data is permanently removed when TTL reaches zero.
+
+- **Contract instances**:
+  - The contract Wasm or instance itself can be archived when their TTL reaches zero, making them inaccessible until restored.
+  - `Instance` data is archived when the contract instance is archived.
+
+### What can be restored?
+
+- **Contract data**:
+
+  - `Persistent` and `Instance` data entries can be restored after being archived.
+  - `Temporary` data cannot be restored once removed.
+
+- **Contract instances**:
+  - Archived contract Wasm or instances can be restored, making them accessible again.
+  - `Instance` data is automatically restored when the contract instance is restored.
+
+See this [guide](/docs/learn/encyclopedia/storage/state-archival) for advanced details on state archival.
```

Comment:
Isn't everything up to this point already described at length in your [cross reference](https://developers.stellar.org/docs/learn/encyclopedia/storage/state-archival)? It'd be more valuable if we focused on the client side instead of repeating things.